tests/test_butlerUtils.py

```diff
@@ -119,9 +119,9 @@ def test_RECENT_DAY(self):
         self.assertTrue(RECENT_DAY <= todayInt)  # in the past
         self.assertTrue(RECENT_DAY >= 20200101)  # not too far in the past
 
-        # check that the value of RECENT_DAY has data
+        # check that the value of RECENT_DAY is before the end of the data.
         daysWithData = getDaysWithData(self.butler)
-        self.assertIn(RECENT_DAY, daysWithData)
+        self.assertLess(RECENT_DAY, max(daysWithData))
```

Comment:
Oooh, that's a very good catch. So, initially I thought that you were right, but now that I think about it, the way it's used, it goes in `where` clauses with a regular `>` sign, so I think it actually _does_ need to be strictly less than, otherwise it would exclude _all_ the data if `RECENT_DAY` is set to the current day_obs. That shouldn't happen, because the update function does `RECENT_DAY = max(day-1, RECENT_DAY)`, but I'm still a little uncomfortable.

Reply:
What do you think about me changing this to be `assertLessEqual`, and changing all the uses of `RECENT_DAY` in the `where`s to be `>=`? That feels safer and preferable, to me, but am curious if you have thoughts.

Reply:
Pushed a commit to those ends, I think I'm pretty happy with that, and tests are OK with that too.